package.json

```diff
@@ -1,33 +1,78 @@
 {
-  "name": "react-searchbox",
-  "version": "0.1.0",
-  "private": true,
-
-  "devDependencies": {
-    "prettier": "^1.18.2",
-    "eslint-plugin-react-hooks": "^2.0.1"
+  "name": "react-searchbox-zzzzz",
+  "version": "2.0.4",
+  "description": "React library kit based on ReactiveSearch's DataSearch",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/appbaseio/react-searchbox.git"
   },
-  "dependencies": {
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-scripts": "3.1.2"
+  "author": "Anuj Shah <anujshah584@gmail.com>",
+  "bugs": {
+    "url": "https://github.com/appbaseio/react-searchbox/issues"
+  },
+  "homepage": "https://github.com/appbaseio/react-searchbox#readme",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "jsnext:main": "dist/index.es.js",
+  "engines": {
+    "node": ">=8",
+    "npm": ">=5"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "test": "cross-env CI=1 react-scripts test --env=jsdom",
+    "test:watch": "react-scripts test --env=jsdom",
+    "build": "rollup -c",
+    "start": "rollup -c -w",
+    "prepare": "yarn run build",
+    "predeploy": "cd example && yarn install && yarn run build",
+    "deploy": "gh-pages -d example/build"
+  },
+  "peerDependencies": {
+    "prop-types": "^15.5.4",
+    "react": "^15.0.0 || ^16.0.0",
+    "react-dom": "^15.0.0 || ^16.0.0"
+  },
+  "dependencies": {
+    "downshift": "^3.3.1",
+    "@emotion/core": "^10.0.17",
+    "@emotion/styled": "^10.0.17",
+    "react-highlight-words": "^0.16.0",
+    "@appbaseio/searchbase": "^1.0.0-alpha",
+    "cross-fetch": "^3.0.4"
```

Comment:
What's the use of `cross-fetch` ? We are handling all request from `Searchbase`. Right ?